pay-api/migrations/versions/4c3b8d61d200_remove_unused_table.py

```diff
@@ -0,0 +1,82 @@
+"""remove_unused_table
```

Comment:
This is to remove unused table which we kept earlier as a backup in case if our migration fails